worlds/ladx/ItemIconGuessing.py

```diff
@@ -67,6 +67,7 @@
     'WAND': 'MAGIC_ROD',
 
     # PEGASUS_BOOTS
+    'BOOST': 'PEGASUS_BOOTS',
```

Comment:
Would recommend against this one. Lots of KH2 items have "Boost" in the name, like Combo Boost or "Item Boost" which have nothing to do with boots